generators/server/templates/src/test/java/package/web/rest/TestUtil.java.ejs

```diff
@@ -265,15 +261,15 @@ public final class TestUtil {
 <%_ } _%>
 <%_ if (searchEngine === 'couchbase') { _%>
 
-    public static <T> void retryUntilNotEmpty(Callable<Iterable<T>> iterableCallable) {
```

Comment:
Can be replaced with Reactor version:
```java
public static <T> void retryUntilNotEmpty(Callable<Iterable<T>> iterableCallable) {
        Mono.defer(
                () -> Mono.fromCallable(iterableCallable)
                    .flatMapIterable(Function.identity())
                    .next()
                    .switchIfEmpty(Mono.error(new NoSuchElementException()))
            )
            .retry()
            .timeout(Duration.ofSeconds(5))
            .block();
    }
```

Reply:
Also please use derived Constants : `searchEngineCouchbase`

Reply:
I've added that function to the code